src/components/Login.jsx

```diff
@@ -45,7 +45,7 @@ function Login({
       </Form.Group>
       <Button
         className='width-50'
-        block
+        block='true'
```

Comment:
the string true?

Reply:
Was getting a weird error in the console about it not supposed to be a boolean... just removed it and it seems to be working fine though? I'll leave it deleted